apps/src/scratch/scratch.js

```diff
@@ -0,0 +1,193 @@
+import AudioEngine from 'scratch-audio';
+import Renderer from 'scratch-render';
+import Storage from 'scratch-storage';
+import VM from 'scratch-vm';
+import Blockly from 'scratch-blocks/dist/vertical.js';
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+import { getStore, registerReducers } from '@cdo/apps/redux';
+import * as commonReducers from '@cdo/apps/redux/commonReducers';
+import { singleton as studioApp } from '@cdo/apps/StudioApp';
+import ScratchView from './ScratchView';
+import { scratchDefaultProject } from './scratchDefaultProject';
+
+const Scratch = window.Scratch = window.Scratch || {};
+
+export default function init(options) {
+  registerReducers(commonReducers);
+
+  options.maxVisualizationWidth = 480;
+  options.vizAspectRatio = 4 / 3;
+  options.enableShowCode = false;
+  options.pinWorkspaceToBottom = true;
+  options.skin = {};
+  window.appOptions = options;
+
+  studioApp().configure(options);
+  studioApp().setPageConstants(options, {});
+
+  ReactDOM.render(
+    <Provider store={getStore()}>
+      <ScratchView onMount={onMount} />
+    </Provider>,
+    document.getElementById(options.containerId),
+  );
+
+  function onMount() {
+    studioApp().init(options);
+  }
```

Comment:
nit: Simple enough function that it might be easier to read if you just inline in JSX, i.e.
```jsx
<ScratchView onMount={() => studioApp().init(options)}/>
```